### PR TITLE
frp, frpc, frps: add page

### PR DESCRIPTION
**Pull request by reinhart1010**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 0.58.1

Though still in alpha, [frp](github.com/fatedier/frp) is already stable enough to create reverse proxies for HTTP, HTTPS, and other TCP-based (e.g. SSH, RDP) connections.

### CODE REVIEW

File: pages/common/frpc.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`frpc`
+
+- Start the service, using the newer TOML config file (`frps.toml` instead of `frps.ini`) in the current directory:
+
+`frpc --config ./frps.toml`
+
+- Start the service, using a specified config file:
+
+`frpc --config {{path/to/file}}`
```
[the remark this answers is not in the document]
Comment (reinhart1010) [reply pages/common/frpc.md:17]: Should I treat the same with `{{-v|--version}}` ?

[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/frpc.md:17]: IG it's fine to have the same syntax for those examples.

File: pages/common/frpc.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`frpc verify --config {{path/to/file}}`
+
+- Generate `frpc` command autocompletion setup script for Bash, Fish, PowerShell, or Zsh (printed to `stdout`):
+
+`frpc completion {{bash|fish|powershell|zsh}}`
+
+- Display version:
+
+`frpc --version`
```
Comment (spageektti) [inline pages/common/frpc.md:29]:
```suggestion
`frpc {{-v|--version}}`
```

File: pages/common/frps.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`frps verify --config {{path/to/file}}`
+
+- Generate `frps` command autocompletion setup script for Bash, Fish, PowerShell, or Zsh (printed to `stdout`):
+
+`frps completion {{bash|fish|powershell|zsh}}`
+
+- Display version:
+
+`frps --version`
```
Comment (spageektti) [inline pages/common/frps.md:29]:
```suggestion
`frps {{-v|--version}}`
```

File: pages/common/frpc.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Start the service, using a specified config file:
+
+`frpc --config {{path/to/file}}`
+
+- Verify that the configuration file is valid:
+
+`frpc verify --config {{path/to/file}}`
+
+- Generate `frpc` command autocompletion setup script for Bash, Fish, PowerShell, or Zsh (printed to `stdout`):
```
Comment (vitorhcl) [inline pages/common/frpc.md:23]:
```suggestion
- Print autocompletion setup script for Bash, Fish, PowerShell, or Zsh:
```

What about this?

File: pages/common/frpc.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- Start the service, using the newer TOML config file (`frps.toml` instead of `frps.ini`) in the current directory:
+
+`frpc --config ./frps.toml`
+
+- Start the service, using a specified config file:
+
+`frpc --config {{path/to/file}}`
+
+- Verify that the configuration file is valid:
```
Comment (vitorhcl) [inline pages/common/frpc.md:19]:
```suggestion
- Check if the configuration file is valid:
```

File: pages/common/frps.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+- Start the service, using a specified config file:
+
+`frps --config {{path/to/file}}`
+
+- Verify that the configuration file is valid:
+
+`frps verify --config {{path/to/file}}`
+
+- Generate `frps` command autocompletion setup script for Bash, Fish, PowerShell, or Zsh (printed to `stdout`):
```
Comment (vitorhcl) [inline pages/common/frps.md:23]:
```suggestion
- Print autocompletion setup script for Bash, Fish, PowerShell, or Zsh:
```

File: pages/common/frps.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+- Start the service, using the newer TOML config file (`frps.toml` instead of `frps.ini`) in the current directory:
+
+`frps --config ./frps.toml`
+
+- Start the service, using a specified config file:
+
+`frps --config {{path/to/file}}`
+
+- Verify that the configuration file is valid:
```
Comment (vitorhcl) [inline pages/common/frps.md:19]:
```suggestion
- Check if the configuration file is valid:
```

**Comment by spageektti** [comment]
Hi @reinhart1010. Can you please check the suggestions?

**Review by gutjuri — APPROVED** [review]

**Pull request merged**